lib/go-atscfg/meta.go

```diff
@@ -40,6 +41,23 @@ type ConfigProfileParams struct {
 // TODO change the config system to not use old API paths, and remove this.
 const APIVersion = "2.0"
 
+// RequiredFiles is a constant (because Go doesn't allow const slices).
```

Comment:
nit, but it's `requiredFiles`, not `RequiredFiles`.

Reply:
Fixed